src/views/AuthenticationWrapper/DevSecurity.tsx

```diff
@@ -117,18 +130,19 @@ const DevSecurity = ({ children }: ParentComponentProps) => {
         style={{ display: 'inline-block', border: 'solid 1px orangered' }}
       >
         <legend>Select job codes</legend>
-        {jobCodes.map(code => (
-          <p key={code}>
+        {Object.keys(groups).map(group => (
+          <Fragment key={group}>
```

Comment:
I got the feeling that we wrapped this in `p` for `display: block` and spacing. Even though this is an internal component that end users wont use, I wanted to flag this and get into the habit of having good HTML. Having the `p` might be confusing to screen reader users because `p` should be for paragraphs and I don't think checkboxes should be in them.